platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/testapp/maps/widgets/CompassViewTest.java

```diff
@@ -40,19 +38,21 @@ public void testDefault() {
   }
 
   @Test
+  @Ignore
   public void testVisible() {
```

Comment:
I think `testVisible` is valid method name for a test that is testing if something is visible. In any case it seems `testVisible` matches the standard conventions more than `checksVisibility`:

```
Naming a Method

Although a method name can be any legal identifier, code conventions restrict method names. By convention, method names should be a verb in lowercase or a multi-word name that begins with a verb in lowercase, followed by adjectives, nouns, etc. In multi-word names, the first letter of each of the second and following words should be capitalized. Here are some examples:

run
runFast
getBackground
getFinalData
compareTo
setX
isEmpty

Typically, a method has a unique name within its class. However, a method might have the same name as other methods due to method overloading.
```

I'm not seeing any runsFast or comparesTo or setsX in that example list.